package.json

```diff
@@ -82,6 +82,7 @@
     "react-dropzone": "4.3.0",
     "react-final-form": "3.6.5",
     "react-final-form-arrays": "1.0.6",
+    "react-ga": "3.3.1",
```

Comment:
@awalkowiak I believe the GA release supported by this plugin will be turned off July 1. Should we use the following instead?

- https://www.npmjs.com/package/react-ga4
- https://support.google.com/analytics/answer/11583528

Reply:
@mdpearson yeah I am aware of this. I added the "next step / to do" to the ticket (https://pihemr.atlassian.net/browse/OBPIH-5218?focusedCommentId=140355) to replace it with the ga4 version when we will be forced to do it because what I understood from @jmiranda's comment - we want to wait until it "will be ~forced on~ done by google for us" (https://pihemr.atlassian.net/browse/OBPIH-5218?focusedCommentId=140307). Replacing the old react-ga with the new one will be easy in that case.

I will recheck tomorrow morning if react-ga4 is backward compatible with od Universal Analytics tracking id. If it is then I'll swap the lib in this PR.

Reply:
I'll investigate it further.